doc/man/scons.xml

```diff
@@ -6508,52 +6470,43 @@ a dictionary that maps keys
 (typically the file suffix, but see below for more discussion)
 to other Scanners that should be called.
 </para></listitem>
-</orderedlist>
+</itemizedlist>
 
 <para>If the argument is a Python function,
 the function must accept three required arguments
 and an optional fourth:
-<literal>def scanner_function(<parameter>node</parameter>, <parameter>env</parameter>, <parameter>path</parameter>, <parameter>arg=None</parameter>):</literal>
 </para>
 
-<para>The
-<parameter>node</parameter>
-argument is the internal
-&SCons; node representing the file.
-Use
-<literal>str(<replaceable>node</replaceable>)</literal>
-to fetch the name of the file, and
-<literal><replaceable>node</replaceable>.get_contents()</literal>
-to fetch contents of the file.
-Note that the file is
-<emphasis>not</emphasis>
-guaranteed to exist before the scanner is called,
-so the scanner function should check that
-if there's any chance that the scanned file
-might not exist
-(for example, if it's built from other files).</para>
+  <simplelist type="vert">
+  <member><parameter>node</parameter> -
+    The internal &SCons; node representing the file.
+    Use <function>str</function>(<parameter>node</parameter>)
+    to fetch the name of the file, and
+    <replaceable>node</replaceable>.<function>get_contents</function>()
+    to fetch the contents of the file.
```

Comment:
Should probably add that this will return bytes. and get_text_contents() will return a string with the contents.

Reply:
Added

Reply:
I don't see bytes here.. ?